tools/sort_actor_funcs.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+import argparse
+
+parser = argparse.ArgumentParser()
```

Comment:
A small description would be nice.
```suggestion
parser = argparse.ArgumentParser(description='something here')
```